src/table/components/virtualized-table/TableContainer.tsx

```diff
@@ -85,4 +85,9 @@ export default function TableContainer(props: TableContainerProps) {
       </FullSizeContainer>
     </div>
   );
-}
+};
+
+// Export non memoized version for testing purpose
+export { TableContainer };
+
+export default memo(TableContainer);
```

Comment:
On the initial render the `Wrapper` component got an extra render due to the `keyboard` prop being updated. For which the was no reason to re-render so added the memo.

Reply:
nice that you are adding memo very intentionally